pmoves/services/flute-gateway/bpm_encoder_worker.py

```
@@ -0,0 +1,199 @@
+    if user and password:
+        return f"nats://{user}:{password}@{host}:{port}"
+    return f"nats://{host}:{port}"
+
+
+def _redact_url(url: str) -> str:
+    try:
+        parsed = urlparse(url)
+        if parsed.password:
+            return url.replace(parsed.password, "***")
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)